composer.json

```diff
@@ -6,29 +6,34 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
+        "php": ">=7.4",
         "cweagans/composer-patches": "^1.7",
         "drupal/core": "^8.9 || ^9.1",
         "openeuropa/oe_bootstrap_theme": "0.1.202202072010"
     },
     "require-dev": {
         "composer/installers": "^1.11",
-        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
-        "drupal/config_devel": "~1.2",
         "drupal/better_exposed_filters": "^5.0",
+        "drupal/config_devel": "~1.2",
+        "drupal/core-composer-scaffold": "^8.9 || ^9.1",
+        "drupal/ctools": "^3.7",
         "drupal/drupal-extension": "~4.1",
+        "drupal/file_link": "^2.0.4",
```

Comment:
why is needed?
- file_link, pathauto, email_validator?

Reply:
This issue is due to when installing the lowest versions:
`Downgrading drupal/core (9.2.12 => 9.2.0): Loading from cache`
these modules do not have the core_version_requirement key on their info file.
An explanation to each is on the _readme key on composer.json
`
"_readme": [
            "Explicit minimum version requirement of drupal/ctools module due to D9.2 compatability.",
            "Explicit requirement for drupal/file_link due to https://www.drupal.org/project/file_link/issues/3147517. It can be removed when oe_media requires version 2.0.4 or above.",
            "Explicit requirement for drupal/pathauto due to D9.2 compatability according to https://www.drupal.org/node/2979476.",
            "Explicit requirement for egulias/email-validator due to https://www.drupal.org/project/drupal/issues/3061074#comment-14300579. It can be removed when Drupal core 9.2 support is droppped."
        ],
`